docs/developers/nft-tokens.md

```diff
@@ -396,6 +408,51 @@ StopNFTCreationTransaction {
 }
 ```
 
+### **Change NFT Attributes**
+
+A user that has the `ESDTRoleNFTUpdateAttributes` role set for a given ESDT, can change the attributes of a given NFT/SFT.
```

Comment:
An* user that has the `ESDTRoleNFTUpdateAttributes` role set for a given NFT/SFT, can change its attributes.